src/Cli/dotnet/commands/dotnet-workload/WorkloadInfoHelper.cs

```diff
@@ -56,7 +56,8 @@ internal class WorkloadInfoHelper : IWorkloadInfoHelper
                 userProfileDir,
                 verifySignatures ?? !SignCheck.IsDotNetSigned(),
                 restoreActionConfig: restoreConfig,
-                elevationRequired: false);
+                elevationRequired: false,
+                shouldLog: false);
```

Comment:
This is the main way we get the benefit of the NullLogger - `dotnet --info` and `dotnet workload list` use this to get installed workload state and so we can silence both paths in one go.